app/src/molecules/InterventionModal/index.tsx

```diff
@@ -157,14 +159,13 @@ export function InterventionModal({
             {...headerStyle}
             backgroundColor={headerColor}
             justifyContent={headerJustifyContent}
-            onClick={iconHeadingOnClick}
           >
             {titleHeading}
-            <Flex alignItems={ALIGN_CENTER} gridGap={SPACING.spacing12}>
+            <Flex alignItems={ALIGN_CENTER} onClick={iconHeadingOnClick}>
               {iconName != null ? (
-                <Icon name={iconName} size={SPACING.spacing32} />
+                <Icon name={iconName} css={ICON_STYLE} />
               ) : null}
-              {iconHeading != null ? iconHeading : null}
+              {iconHeading ?? null}
```

Comment:
I prefer this personally, but we try to keep the same format across the codebase:
```suggestion
              {iconHeading != null ? iconHeading : null}
```